src/flows/components/Sidebar.tsx

```diff
@@ -191,24 +205,27 @@ const Sidebar: FC = () => {
               testID={`${title}--list-item`}
             >
               {title}
-            </List.Item>
+            </DropdownItem>
           )
         })
 
       const sectionTitle =
         typeof section.title === 'function' ? section.title() : section.title
-      return (
-        <div key={sectionTitle}>
-          <List.Divider text={`${sectionTitle}:`} />
-          {links}
-        </div>
-      )
+      return <div key={sectionTitle}>{links}</div>
     })
 
   return (
-    <div className="flow-sidebar">
-      <List>{sections}</List>
-    </div>
+    <DropdownMenu
+      style={{
+        width: '200px',
+        position: 'absolute',
+        top: '40px',
+        zIndex: 2,
+        padding: '10px 0px',
+      }}
```

Comment:
can this all be a class with cf variables? Why the absolute positioning? Is it absolute relative to a panel?

Reply:
Absolute is necessary for it to be positioned the way it is relative to the panel (pun intended). This isn't really related to clockface positioning or styling, so i didn't use clockface variables, but i can move this into a css file, for sure

Reply:
gotcha.